src/pages/CreatePool/CreatePoolDialog.tsx

```diff
@@ -101,8 +101,20 @@ export default function ReviewCreatePool({
       }
       resetFields()
     } catch (err) {
-      console.error(err)
-      enqueueToast("error", "Unable to deploy Permissionless Pool")
+      console.error({ err })
+      const er = err as { error: { data: { message: string } } }
```

Comment:
Is there a type defined somewhere that we could use?

Reply:
i couldnt find one